From ed4d0de09d65abe2b3a2c237b9713591328ac423 Mon Sep 17 00:00:00 2001
From: Don <don.j.olmstead@gmail.com>
Date: Sat, 24 Sep 2022 18:57:06 -0700
Subject: [PATCH 3/3] Remove icu-uc from pkgconfig

The icu build doesn't output a pkgconfig so vcpkg will fail to fix up harfbuzz's pkgconfig.
---
 src/harfbuzz-icu.pc.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/harfbuzz-icu.pc.in b/src/harfbuzz-icu.pc.in
index 949869a35..5be5b2686 100644
--- a/src/harfbuzz-icu.pc.in
+++ b/src/harfbuzz-icu.pc.in
@@ -8,6 +8,5 @@ Description: HarfBuzz text shaping library ICU integration
 Version: %VERSION%
 
 Requires: harfbuzz
-Requires.private: icu-uc
 Libs: -L${libdir} -lharfbuzz-icu
 Cflags: -I${includedir}/harfbuzz
-- 
2.41.0.windows.1

